configure.ac

```diff
@@ -45,6 +45,10 @@ AS_IF([test "x$with_folly" != "x"], [
 	)
 ])
 
+dnl Openssl headers are required.
+AC_CHECK_HEADER([openssl/sha.h], [], [
+  AC_MSG_FAILURE([openssl/sha.h is required.  Try e.g. "dnf install openssl-devel"])
```

Comment:
Can you make these instructions refer to the installation docs instead, and then expand on the various platform specific ways to install the required packages there?

You can find info about updating the docs here: https://facebook.github.io/watchman/contributing.html#dont-forget-the-docs